R/build_episode.R

```diff
@@ -241,7 +241,9 @@ build_episode_md <- function(path, hash = NULL, outdir = path_built(path),
     spinner = sho,
     env = c(callr::rcmd_safe_env(),
       "RENV_PROFILE" = profile,
-      "RENV_CONFIG_CACHE_SYMLINKS" = renv_cache_available())
+      "RENV_CONFIG_CACHE_SYMLINKS" = renv_cache_available(),
+      "LANGUAGE" = get_config()$lang,
```

Comment:
I'm not completely sure if we should rely on `get_config()`'s default arg here, or provide another value?

I'd welcome any insights on this, or I will investigate more myself once #644 have been decided.